ql/src/test/results/clientpositive/llap/materialized_view_create_rewrite_5.q.out

```diff
@@ -404,7 +404,7 @@ Table Parameters:
 	numFiles            	2                   
 	numRows             	10                  
 	rawDataSize         	0                   
-	totalSize           	#Masked#
+	totalSize           	#Masked#                
```

Comment:
Need to regenerate this out once remove the `totalSize`.

Reply:
Done, thanks